### DISCUSSION TITLE
Is fontawesome still used?

### DISCUSSION THREAD

**Discussion by jmenzies12** [eclipse-glsp/glsp#1407] [Q&A] [answered]
Hey,

Just wanted to check if fontawesome is still used? I see it in the vscode-integration example, but I also see an old issue that was swapping it out for codicons. Just checking so that I can update my webview code and remove it if it is not used.

Cheers
Jonathan

**Comment by tortmayr** [comment, marked as answer]
Hi,
no fontawesome is no longer used in the base framework. We migrated to codicons a while ago.
Feel free to remove it from your webview code.
